_content/av/can-everyone-do-the-full-lotus-pose_akram-yoga.md

```diff
@@ -0,0 +1,19 @@
+---
+title: "Can everyone do the full lotus pose?"
+authors:
+  - "Akram Yoga & Teacher Training
+external_url: "https://www.youtube.com/watch?v=FfAJKjn2CjY"
+drive_links:
+  - "https://docs.google.com/document/d/1zok4vrUg_cC5gX10PTrVJDaUspwOSdzYCaK7lIN2NjE/edit?usp=drive_link"
+course: problems
+tags:
+  - yoga
+  - asana
+year: 2021
+month: jan
+minutes: 9
+---
+
+> If you have the type of anatomy that isn't suited to lotus pose, time and effort won't change a thing. There is a saying in yoga that "practice and all is coming". In this context all that will be coming is pain.
+
+A short video explaining the anatomy of the lotus pose and why everyone cannot meditate in this posture.
```

Comment:
You meant `why some people` or `why not everyone can`... What you wrote means that nobody can do lotus pose!😉

The book "Several Short Sentences About Writing" ends in a long series of sentences which didn't mean what their writers intended them to. I do recommend the book, as it was an inspiring and instructive set of good and bad sentences for me.